redash/handlers/groups.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from redash.handlers.base import BaseResource
+from redash.wsgi import api
+from flask import make_response
+from redash import utils
+from redash.models import Group
+
+class GroupsAPI(BaseResource):
+    def get(self):
+        return self.make_json_response(Group.select())
```

Comment:
Why not just return the groups array? flask-restful takes care of serialization. 

Also, please return the groups as an array, instead of a property in an object to remain consistent with the other APIs.